docs/include/conf/cincinnati/visiting.rst

```diff
@@ -0,0 +1,69 @@
+
+Eating
+------
+
+Snacks and drinks are provided throughout the conference. For meals during the conference, connect with new friends and explore one of Cincinnati's most diverse neighborhoods. Here are some resources that will help you find something good to eat:
+
+- The `Over-the-Rhine Chamber of Commerce <http://business.otrchamber.com/list/ql/restaurants-food-beverages-22?_ga=2.247727175.1927584494.1524751496-300914214.1524751496>`_ lists a number of restaurants and bars in the neighborhood
+- `Open Table <https://www.opentable.com/n/cincinnati/over-the-rhine-restaurants>`_ also lists great restaurants in Over-the-Rhine
+
+Where to stay
+-------------
+
+We are not working with any of the hotels for attendee accommodations, but there are many hotels available within walking distance of the venue:
+
+- `Symphony Hotel <http://www.symphonyhotel.com>`_
+- `The Cincinnatian Hotel <http://www.cincinnatianhotel.com>`_
+- `21c Museum Hotel <https://www.21cmuseumhotels.com/cincinnati/>`_
+- `Millennium Cincinnati <https://www.millenniumhotels.com/en/cincinnati/millennium-hotel-cincinnati/>`_
+- `Hilton Cincinnati Netherland Plaza <http://www3.hilton.com/en/hotels/ohio/hilton-cincinnati-netherland-plaza-CVGNPHF/index.html>`_
+
+Check TripAdvisor to find other hotels and accommodations in the Cincinnati area.
+
+How to get around
+-----------------
+
+Cincinnati is a small, walkable city with public transportation options available. Over-the-Rhine is about 1 mile north of the downtown area, which is bordered to the south by the Ohio River. We encourage you to connect with other documnentarians and explore during your stay.
+
+Streetcars and Buses
+~~~~~~~~~~~~~~~~~~~~
+
+Cincinnati has a `streetcar <http://www.cincinnatibellconnector.com>`_ that makes a loop from `Findlay Market <http://findlaymarket.org>`_ (north of Over-the-Rhine) to the `Great American Ball Park <https://www.mlb.com/reds/ballpark>`_ (downtown, on the Ohio River). You can also use Cincinnati's `bus network <http://www.go-metro.com/index.php>`_, which also connects to routes in northern Kentucky, just across the Ohio River.
+
+Taxi Companies
+~~~~~~~~~~~~~~
+
+Cincinnati has both Lyft and Uber. If you prefer local taxi companies, here is a list of `Cincinnati's licensed taxicab companies <https://www.cincinnati-oh.gov/public-services/linkservid/1831121A-DAD6-E032-9E01ECEF5B91DF25/showMeta/0/>`_.
+
+Please note that it is *not* common to hail taxis in Cincinnati. Your best bet is to call in advance.
+
+Driving to Union Hall
+~~~~~~~~~~~~~~~~~~~~~
+If you drive to the conference, parking in downtown Cincinnati is plentiful and inexpensive on the weekends. Use the `Over-the-Rhine Chamber of Commerce <http://otrchamber.com/pages/WheretoPark>`_ or `3CDC Parking <https://www.3cdc.org/where-to-park/>`_ sites to find parking facilities.
```

Comment:
Probably worth pointing out the Mercer Garage is by far the closest, literally across the street.

Reply:
Thank you -- I added that info with a link to it on Google Maps.